### PR TITLE
RELEASE 0.8.1

### PR DESCRIPTION
**Pull request by francisco-dlp**
TODO
--------

* [x] Authors verify that thier contributions appear in CHANGES
* [x] Authors verify that the issues related to their contributions have the right labels in GitHub
* [x] Authors verify that their contributions are well explained in the header of the corresponding GitHub issues.
* [x] Change release version
* [x] Update change list
* [ ] Add release tag

### CODE REVIEW

**Comment by vidartf** [comment]
I've looked over and fixed my things (where I'm listed as author).

**Comment by jat255** [comment]
My things look alright as well, I think.

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,57 @@ We only cover here the main highlights, for a detailed list of all the changes
...
+This is a maintenance release. Follow the following links for details on all
+the `bugs fixed
+<https://github.com/hyperspy/hyperspy/issues?page=1&q=is%3Aclosed+milestone%3A0.8.1+label%3Abug&utf8=%E2%9C%93>`_,
+`feature
+<https://github.com/hyperspy/hyperspy/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A0.8.1++label%3Aenhancement+>`_
+and `documentation
+<https://github.com/hyperspy/hyperspy/issues?q=is%3Aclosed+milestone%3A0.8.1+label%3Adocumentation>`_ enhancements.
+
+It also includes some new features and introduces important API changes that
+will be fully enforced in HyperSpy 0.9.
```
Comment (vidartf) [inline CHANGES.txt]: Note that the old `create_model` and `unfold_if_multidim` no longer works, so they are actually fully enforced already. As this is a maintenance build, we might want to consider 'reinstating' the functionality while keeping the warnings.

Comment (francisco-dlp) [inline CHANGES.txt]: ``create_model`` is still there.  ``unfold_if_multidim`` is not. +1 to put it back with a VisibleDeprecationWarning. I'll do it.

**Pull request merged**